### DISCUSSION TITLE
Vercel Nodejs backend deployment - This Serverless Function has crashed.

### DISCUSSION THREAD

**Discussion by danielcornejodev** [vercel/community#5856] [Help]
### Summary

I haven't been able to solve this Vercel 500 error. Appears I am using compatible versions of mongodb and mongoose

### Steps to Reproduce

Cannot find module '../sdam/server_selection'
Require stack:
- /var/task/node_modules/mongodb/lib/operations/execute_operation.js
- /var/task/node_modules/mongodb/lib/admin.js
- /var/task/node_modules/mongodb/lib/index.js
- /var/task/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js
- /var/task/node_modules/mongoose/lib/drivers/node-mongodb-native/index.js
- /var/task/node_modules/mongoose/lib/index.js
- /var/task/node_modules/mongoose/index.js
- /var/task/db/index.js
- /var/task/app.js
- /var/task/___now_launcher.js
- /var/runtime/index.mjs
Did you forget to add it to "dependencies" in `package.json`?
INIT_REPORT Init Duration: 202.85 ms	Phase: invoke	Status: error	Error Type: Runtime.ExitError
Error: Runtime exited with error: exit status 1
Runtime.ExitError

package.json: 

{
  "name": "stay-fit-app-server-server",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "start": "node server.js",
    "dev": "nodemon server.js"
  },
  "dependencies": {
    "bcrypt": "^5.1.0",
    "cookie-parser": "^1.4.6",
    "cors": "^2.8.5",
    "dotenv": "^16.3.1",
    "express": "^4.18.2",
    "express-jwt": "^8.4.1",
    "express-session": "^1.17.3",
    "jsonwebtoken": "^9.0.1",
    "mongodb": "^6.0.0",
    "mongoose": "^8.1.3",
    "morgan": "^1.10.0"
  },
  "devDependencies": {
    "nodemon": "^3.0.1"
  }
}

**Comment by bhanugurgulwar** [comment]
Getting Same Error, if anyone have solution

**Comment by Keyur200** [comment]
Same error

**Comment by BulletStormXT** [comment]
getting the same error too

**Comment by dayanapanova** [comment]
Same error

[a comment by an automated account is not shown]
